### PR TITLE
docs: add license badge (FOSSA)

### PR DESCRIPTION
**Pull request by ccamel**
Add the license badge provided by [FOSSA](https://fossa.com/):

[FOSSA Status](https://app.fossa.com/reports/e5217c6b-d64c-49c0-a3ce-a74484a9854c)

### CODE REVIEW

**Review by ErikssonJoakim — APPROVED** [review]

**Review by hleonardtek — APPROVED** [review]

**Review by ad2ien — APPROVED** [review]
:paw_prints:

**Review by JoeyDjr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 [![Build](https://github.com/okp4/ui/actions/workflows/build.yml/badge.svg)](https://github.com/okp4/ui/actions/workflows/build.yml)
 [![Maintainability](https://api.codeclimate.com/v1/badges/696cb4f6326aebdc7abe/maintainability)](https://codeclimate.com/github/okp4/ui/maintainability)
 [![Test Coverage](https://codecov.io/gh/okp4/ui/branch/main/graph/badge.svg?token=WHLU66SCC2)](https://codecov.io/gh/okp4/ui)
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fokp4%2Fui.svg?type=shield)](https://app.fossa.com/reports/e5217c6b-d64c-49c0-a3ce-a74484a9854c)
+
```
Comment (lolottetheclash) [inline README.md:13]: I'm not sur this is really important but there is an extra empty line here

**Review by lolottetheclash — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Review by ingvaar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ccamel) [reply README.md:13]: Yes, the goal is to have a carriage return between the two blocks of badges. 😌

**Review by amimart — APPROVED** [review]
👍

**Pull request merged**